Add POI create to testing

donelleharris opened the pull request.

michaelmeyer26 committed:
Added a form onto the points/add.html page

michaelmeyer26 committed:
feat: initial implementation of allowing user to add a POI

donelleharris committed:
Move styling and scripts from main pages to partials and .js files

donelleharris committed:
Add profile image placeholder to profile page

donelleharris committed:
Add styling to user profile image for size and shape

donelleharris committed:
Add security scripting to head of individual views

donelleharris committed:
Add header and link to routes/index on landing page

RMK1775 committed:
Add route create page

Jason-Carrillo committed:
Merge branch 'donelle-user-profile-page' of github.com:Bio-JRMD-Capstone/Sidecar into donelle-user-profile-page

RMK1775 committed:
Adds Donelles css package

RMK1775 committed:
Add mapping for route create page

RMK1775 committed:
Amend pom file for initiating deployment

donelleharris committed:
Add filestack.js, scripts & html code for user image upload

donelleharris committed:
Add css/sass stying to registration

donelleharris committed:
Add map to routes/create, update links in head

Jason-Carrillo committed:
pulled branch

donelleharris committed:
Correct links to styling and js files in routes/create 

Switch security config to access create w/out login

donelleharris committed:
Add correct lon/lat to js file for centering map

donelleharris committed:
Add styling to login form

Jason-Carrillo committed:
cleaned code

Jason-Carrillo committed:
fixed map dispaly issue

Jason-Carrillo committed:
added remove all markers on creating a route page

donelleharris committed:
Add Sass files to branch

Jason-Carrillo committed:
added display directions function still clearing it up

Jason-Carrillo committed:
commented out broken code, and left only working code for functionaility

donelleharris committed:
Fix issues with connection to css and scripting files

donelleharris committed:
Update css/sass styling library

RMK1775 committed:
Merge pull request #26 from Bio-JRMD-Capstone/donelle-user-profile-page

Update css/sass styling library

RMK1775 committed:
Create Route Create page

RMK1775 committed:
Merge remote-tracking branch 'origin/ryan-page-routecreate' into ryan-page-routecreate

RMK1775 committed:
Update SASS imports

donelleharris committed:
Import additional style sheets

donelleharris committed:
Add images to project

RMK1775 committed:
Fix missing navbar partials

RMK1775 committed:
Merge pull request #27 from Bio-JRMD-Capstone/donelle-user-profile-page

Compare branches and merge

donelleharris committed:
Update sass color variables

michaelmeyer26 committed:
finished the add POI function

donelleharris committed:
Merge pull request #28 from Bio-JRMD-Capstone/michael-add-poi

Michael add poi

Jason-Carrillo committed:
cleaned up code

donelleharris committed:
Update feature-box to include missing color assignments

donelleharris committed:
Update filestack.js & scripting to add images

Jason-Carrillo committed:
cleaned code

donelleharris committed:
Resolving merge conflict w/ testing branch

Jason-Carrillo committed:
fixed constructor for settime

donelleharris committed:
Correct Route time getter

Jason-Carrillo committed:
route now has option to end where it starts

RMK1775 reviewed (changes requested):
This is amazing progress; especially being new tech outside the curriculum.

- Absolutely agree with your decision to set one category to a POI
- Page should require a logged in user
- Is it possible to put in a hard address and have the map zoom in?
- The follow on pages need to be fleshed out to show current POIs

donelleharris committed:
Update link in head for sass

RMK1775 committed:
Merge branch 'testing' into donelle-user-profile-page

RMK1775 committed:
Merge pull request #31 from Bio-JRMD-Capstone/donelle-user-profile-page

Merge styled views into testing

michaelmeyer26 committed:
fixed pulling the wrong value and attaches the logged-in user to the POI when adding it to the db

Jason-Carrillo committed:
pushed array of string coordinates to database

michaelmeyer26 committed:
Fixes the category being displayed incorrectly a point's infowindow

michaelmeyer26 committed:
makes it so that a user needs to be logged in to add a POI

Jason-Carrillo committed:
added lng to the coordiantes that are being pushed ot the markers string array

michaelmeyer26 committed:
fixes the POI name being undefined

Jason-Carrillo committed:
route directions now displays on the page after a user submits a route

michaelmeyer26 committed:
Finished the view for /points

donelleharris committed:
Merge branch 'testing' into michel-all-pois

donelleharris committed:
Merge pull request #33 from Bio-JRMD-Capstone/michel-all-pois

Michael all pois

Jason-Carrillo committed:
cleaned code to display the total distance of a route and display as the value for distance

donelleharris committed:
Merge branch 'testing' into jason

donelleharris committed:
Merge pull request #34 from Bio-JRMD-Capstone/jason

Jason

donelleharris committed:
Fix issues with overlapping elements, margins, padding & stying links in head

donelleharris committed:
Move points and route creation behind security

Jason-Carrillo committed:
added time/duration to be pushed to the value of time on for the route

Jason-Carrillo committed:
made time value readonly

Jason-Carrillo committed:
time and distance shows up correctly now

Jason-Carrillo committed:
changed time typo in header

Jason-Carrillo committed:
values of distance and duration now clear with all info about the route

Jason-Carrillo committed:
after creting a route it now redirects to routes page instead of login

donelleharris committed:
Replace Bootstrap navigation with custom css expanding navigation

donelleharris committed:
Apply css changes to routes/create and poi/create

donelleharris committed:
Merge branch 'donelle-expanding-nav' into donelle-routes-create-test

donelleharris committed:
Merge pull request #35 from Bio-JRMD-Capstone/jasonfix

Jasonfix

donelleharris committed:
Update expandable nav to include link to landing page

donelleharris committed:
Update createRoute.js to most recent version in testing

donelleharris committed:
Add styling and updated map info from testing to minimize merge conflict

RMK1775 committed:
Amend gitignore and associated file

donelleharris committed:
Update stylesheet

donelleharris committed:
Commit auto change in style.css

donelleharris committed:
Add ability to create event

donelleharris committed:
Merge branch 'testing' into donelle-merge-from-testing

donelleharris committed:
Add ability to create new event, post all events to events index view

new event eventType selected by radio button
user may only upload image file

donelleharris committed:
Temporarily switch views back to bootstrap nav

expand-nav logs user out

Jason-Carrillo committed:
all routes display in the routes index page

Jason-Carrillo committed:
individual pages for routes shows up correctly

Jason-Carrillo committed:
changed post to route typo

Jason-Carrillo committed:
changed route page to show username of person who submited it

Jason-Carrillo committed:
route is displayed on the map correctly

donelleharris committed:
Merge pull request #39 from Bio-JRMD-Capstone/jasonfor

Jasonfor

donelleharris committed:
Switch nav from bootstrap to expanding

donelleharris committed:
Add icons to expanding nav

donelleharris committed:
Add styling to events/create

donelleharris committed:
Minor spacing changes for merge to testing

donelleharris committed:
Add all routes to routes/index view

donelleharris committed:
Switch all views to expanding nav

donelleharris committed:
Merge branch 'testing' into donelle-merge-from-testing

donelleharris committed:
Merge branch 'donelle-merge-from-testing' into donelle-user-profile-page

donelleharris committed:
Add user profile edit functionality

User able to edit username, password, email & zip code
Not currently storing or editing photo url from registration

donelleharris committed:
Move edit/delete buttons from routes/index to showRoute views

Jason-Carrillo committed:
Merge pull request #40 from Bio-JRMD-Capstone/donelle-merge-from-testing

Complete styling for Events, Routes & POI create views

Jason-Carrillo committed:
got numbers from the original coordinates to appear by themselves

Jason-Carrillo committed:
got to putting coords into an obj before initializing

Jason-Carrillo committed:
worked through problem

donelleharris committed:
Add user profile image to register and edit views w/ initial styling

Jason-Carrillo committed:
Merge pull request #41 from Bio-JRMD-Capstone/jasonfor

Jasonfor

Jason-Carrillo committed:
added loop to split all coordinates individually

Jason-Carrillo committed:
page now loads the route correctly for each individual route

donelleharris committed:
Merge pull request #42 from Bio-JRMD-Capstone/jasonfix

Jasonfix

donelleharris committed:
Continue progress toward changing user profile pic on users/edit

donelleharris committed:
Merge branch 'testing' into donelle-user-profile-page

donelleharris committed:
Add ability to change user profile image on profile edit view

donelleharris committed:
Change method findAllByUser in Event, POI & Route repositories

allow for list of all routes, events and POIs on user profile view

donelleharris committed:
Add correct navigation to events index, poi add and showRoute views

donelleharris committed:
delete "add" poi view

Jason-Carrillo committed:
added route to display along with directions from each point.

Jason-Carrillo committed:
Merge pull request #43 from Bio-JRMD-Capstone/donelle-user-profile-page

User profile view and edit functionality complete

donelleharris committed:
Merge pull request #44 from Bio-JRMD-Capstone/jasonfix

added route to display along with directions from each point.

donelleharris committed:
Merge branch 'testing' into donelle-user-profile-page

donelleharris committed:
Add ability to create an event with radio button category selection

donelleharris committed:
Add full CRUD functionality for Events

donelleharris committed:
Move edit/delete links to user profile view

donelleharris committed:
Merge branch 'donelle-events' into donelle-user-profile-page

donelleharris committed:
Add delete functionality for routes & poi from user profile view

donelleharris committed:
Adjust layout and styling of show individual route view

donelleharris committed:
Adjust submit button spacing on points/create view

donelleharris committed:
Modify points/create security & category selection

Add category selection to points/create
Add points/create to SecurityConfiguration

donelleharris committed:
Add z-index for styling

donelleharris committed:
Move hidden nav links from left to far right side of views

RMK1775 committed:
Move nav links

- Links no longer overlap other buttons on create pages

donelleharris committed:
Add div for map comments on individual routes page

donelleharris committed:
Merge branch 'donelle-user-profile-page' into donelle-events

RMK1775 committed:
Merge pull request #45 from Bio-JRMD-Capstone/donelle-events

Add full CRUD functionality for Events
